semantic_model_generator/snowflake_utils/snowflake_connector.py

```diff
@@ -180,7 +180,7 @@ def _get_column_representation(
             cursor = conn.cursor(DictCursor)
             assert cursor is not None, "Cursor is unexpectedly None"
             cursor_execute = cursor.execute(
-                f'select distinct "{column_name}" from "{schema_name}"."{table_name}" limit {ndv}'
+                f'select distinct "{column_name}" from {schema_name}.{table_name} limit {ndv}'
```

Comment:
For now we'll remove the quotes since we are fully qualifying the schema name earlier. If we want to support quote identifiers on table and column objects, let's do that as a subsequent effort.